cross-chain/arbitrum/deploy_l2/02_deploy_arbitrum_wormhole_gateway.ts

```diff
@@ -0,0 +1,70 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // These are the fake random addresses for local development purposes only.
+  const fakeTokenBridge = "0x0af5DC16568EFF2d480a43A77E6C409e497FcFb9"
+  const fakeWormholeTBTC = "0xe1F0b28a3518cCeC430d0d86Ea1725e6256b0296"
+
+  const L2TokenBridge = await deployments.getOrNull("TokenBridge") // L2
+  const WormholeTBTC = await deployments.getOrNull("WormholeTBTC") // L2
+  const ArbitrumTBTC = await deployments.get("ArbitrumTBTC")
+
+  let tokenBridgeAddress = ""
+  if (L2TokenBridge && helpers.address.isValid(L2TokenBridge.address)) {
+    log(`using existing L2 TokenBridge at ${L2TokenBridge.address}`)
+    tokenBridgeAddress = L2TokenBridge.address
+  } else {
+    log(`using fake L2 TokenBridge at ${fakeTokenBridge}`)
+    tokenBridgeAddress = fakeTokenBridge
+  }
+
+  let wormholeTBTCAddress = ""
+  if (WormholeTBTC && helpers.address.isValid(WormholeTBTC.address)) {
+    log(`using existing L2 WormholeTBTC at ${WormholeTBTC.address}`)
+    wormholeTBTCAddress = WormholeTBTC.address
+  } else {
+    log(`using fake L2 WormholeTBTC at ${fakeWormholeTBTC}`)
+    wormholeTBTCAddress = fakeWormholeTBTC
+  }
```

Comment:
I'd much prefer to use fake contracts only if the network name is hardhat and otherwise, fail if the contract address could not be resolved. With the current setup, we have a risk that we'll deploy contracts to testnet/mainnet with wrong addresses of Wormhole contracts if something went wrong during the address resolution.

Reply:
Ok, added additional check here [e853dd5](https://github.com/keep-network/tbtc-v2/pull/554/commits/e853dd56a5be6497390badbf4500e9dbb5c8fd46)